CIPs/cip-draft-bls-publickey.md

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
We need to mention gas costs somewhere.
As for the methodology to derive it, one possibility is, paraphrasing from @prestwich:
* We need to choose a "gas per second" number
* Perform benchmarks and set the gas based on that

Another suggestion by @oneeman is relative pricing - take an existing precompile, benchmark your new precompile and the existing one, and price using the ratio.

Reply:
An approach combining them would be to take some existing precompiles and their current gas costs, benchmark them and calculate their implied gas per second numbers, then take the mean or median of those and price using that.

Reply:
I know the matter team has been doing this for various Ethereum precompiles lately. There's a bit of a difference here, as all ethereum precompiles are pure computation while Celo has precompiles that access state (and thus should be considerably more expensive than benching might imply)

Reply:
note: state access costs scale with state size. so any precompile that accesses state should be repriced regularly